pex/bin/pex.py

```diff
@@ -291,6 +291,17 @@ def configure_clp_pex_options(parser):
            'No value (alias for prefer, for backwards compatibility). '
            '[Default: %default]')
 
+  group.add_option(
+      '--include-pyc', '--no-include-pyc',
+      dest='include_pyc',
+      default=True,
+      action='callback',
+      callback=parse_bool,
+      help='Including .pyc files will result in slightly faster startup performance, but at the '
+           'expense of making the generated pex not be reproducible, meaning that if you were to '
```

Comment:
I think this needs to be softened to maybe. If 3.7+ and they have exported SOURCE_DATE_EPOCH, then, today without changes to Pex, they get reproducible .pyc.

Reply:
Done. Although didn't explicitly mention this 3.7+ case because that seems pretty niche. Let me know how the new one is.